### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by SachinRammoorthy**

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi Any changes required?

File: README.md [hunk 1]
```diff
@@ -40,5 +40,9 @@ $ git commit -m "DESCRIBE YOUR CHANGES HERE"
 ```
 $ git push origin
 ```
-**5.** **Send Pull Request** by clicking new pull request button on your repository page
+**5.** **Create a Pull Request** by clicking the _New pull request_ button on your repository page.
 
+
+##Help
```
Comment (aashutoshrathi) [inline README.md]: Give space after `##`

File: README.md [hunk 2]
```diff
@@ -6,29 +6,29 @@
...
 **2.** **Clone** the repository by running following command in [git](https://git-scm.com/):
 ```
 $ git clone https://github.com/[YOUR-USERNAME]/lead-management-android.git
 ```
-**3.** **Open project** using [Android Studio](https://developer.android.com/studio/index.html) by clicking **Open an existing Android Studio Project** 
+**3.** **Open the project** using [Android Studio](https://developer.android.com/studio/index.html) by clicking **Open an existing Android Studio Project** 
 ![](https://lh4.googleusercontent.com/ttV9QNEuOltxmSiZSfRZxDPy_ZetUaBwmm7MeyXTo6xNB8nc6kFAbwU5zWWLaU0AB1xyP8vigMV9Hm7WmJrA=w1863-h978)
 
-**3.** **Run project on AVD** (android virtual device) by clicking **Run** button on Android Studio Toolbar
+**3.** **Run the project on an AVD** (android virtual device) by clicking the **Run** button on Android Studio Toolbar
```
Comment (aashutoshrathi) [inline README.md]: Fix the numbering 😢 
Also, make AVD's full form in Title Case

File: README.md [hunk 3]
```diff
@@ -6,29 +6,29 @@
...
 
 ## Contributing
 
 **1.** Make reasonable changes.
 
-**2.** **Add changes** by running following command in [git](https://git-scm.com/)
+**2.** **Add changes** by running this command on the terminal/command prompt
 ```
 $ git add .
```
Comment (aashutoshrathi) [inline README.md]: This is for adding everything, also add the command to add selected files.

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi done. Please have a look.

**Pull request merged**